.github/workflows/build_images_uv.yaml

```diff
@@ -0,0 +1,71 @@
+name: Docker Image CI
+
+on:
+  pull_request:
+    paths:
+      - "lumigator/**"
+      - ".github/**"
+    branches: ["main"]
```

Comment:
I don't think we need the `branch` section under `pull_request`

Reply:
I leave it because I only want to create new images from main. If someone creates a PR with two different branches (not main) involved, I don't want to generate a new image. WDYT?

Reply:
I'd support that. It's not common but sometimes we will want PRs among non-main branches for some complex features being developed by more than one person.

Reply:
Cool, will change it